github/github.go

```diff
@@ -487,7 +487,7 @@ func (r RateLimits) String() string {
 	return Stringify(r)
 }
 
-// RateLimit is deprecated.  Use RateLimits instead.
+// RateLimit is deprecated.  Deprecated: Use RateLimits instead.
```

Comment:
All right, I've updated it to be more like https://godoc.org/image/jpeg#Reader. It follows the second form as described in https://github.com/golang/go/issues/10909#issuecomment-136492606:

> ```
> 2. long form 1, used with entities lacking docs
> // Deprecated: T is deprecated.
> type T struct {
>      // Deprecated: for backward compatibility FieldC is still populated,
>      // but new code should use FieldD instead.
>      FieldC int32
> 
>      FieldD int64
> }
> ```

PTAL.